src/main/lens-proxy/lens-proxy-certificate.global-override-for-injectable.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import type { SelfSignedCert } from "selfsigned";
+import { getGlobalOverride } from "../../common/test-utils/get-global-override";
+import lensProxyCertificateInjectable from "./lens-proxy-certificate.injectable";
+
+export default getGlobalOverride(lensProxyCertificateInjectable, () => {
+  return {
+    get: () => ({
+      public: "<public-data>",
+      private: "<private-data>",
+      cert: "<ca-data>",
+    }) as SelfSignedCert,
```

Comment:
I think it would be better to not override these or maybe override the runnable instead

Reply:
Why it would be better?

Reply:
The general preference to override as little business logic as possible. That way if a test tries to call `set` multiple times it fails the test instead of succeeding silently.

Reply:
You mean that generating a LensProxy certificate should happen in a runnable and then we override the runnable? I have hard time to see how it would be better (the override would look exactly the same but in a different place) 🤔

Reply:
I would argue that it is better because it more closely represents what happens in the application. If the runnable never gets called then the certificate is not set and should throw and thus causing test failures. Furthermore, if we accidently run the runnable twice then it should also throw.

Reply:
Ok, moved to runnables.